library/MiscUtils.cpp

```diff
@@ -169,14 +169,15 @@ std::string to_search_normalized(const std::string &str)
 }
 
 
-bool word_wrap(std::vector<std::string> *out, const std::string &str,
-               size_t line_length, bool collapse_whitespace)
+bool word_wrap(std::vector<std::string> *out, const std::string &str, size_t line_length,
+               bool collapse_whitespace, bool trim_whitespace_after_newline)
```

Comment:
since this is mutually exclusive with the collapse_whitespace boolean, maybe it would be better expressed as an enum?

Reply:
I made an attempt. Let me know what I need to fix.

Reply:
`word_wrap` itself is not in a namespace, so I think it's fine how you have it. The name of the enum and its values are unique enough that I don't think there is much danger of collision.